v03_pipeline/lib/misc/allele_registry.py

```diff
@@ -159,16 +159,20 @@ def handle_api_response(
             continue
 
         # Extract CAID and allele info
-        caid = allele_response['@id'].split('/')[-1]
-        allele_info = next(
-            record
-            for record in allele_response['genomicAlleles']
-            if record['referenceGenome'] == reference_genome.value
-        )
-        chrom = allele_info['chromosome']
-        pos = allele_info['coordinates'][0]['end']
-        ref = allele_info['coordinates'][0]['referenceAllele']
-        alt = allele_info['coordinates'][0]['allele']
+        try:
+            caid = allele_response['@id'].split('/')[-1]
+            allele_info = next(
+                record
+                for record in allele_response['genomicAlleles']
+                if record['referenceGenome'] == reference_genome.value
+            )
+            chrom = allele_info['chromosome']
+            pos = allele_info['coordinates'][0]['end']
+            ref = allele_info['coordinates'][0]['referenceAllele']
+            alt = allele_info['coordinates'][0]['allele']
+        except (KeyError, StopIteration):
```

Comment:
oof so `referenceGenome` was missing from the response? 

Looking back to https://reg.clinicalgenome.org/doc/AlleleRegistry_1.01.xx_api_v1.pdf, looks like `referenceGenome` in `alleleDefinition` is populated conditionally. 

We could handle it more explicitly but I think a simply try/except is the better option.

Reply:
🤷 yes!  I think this is the laziest answer... if any keys are missing just let it be unmappable.  manual intervention on this is expensive (I think?)